criu/include/mount.h

```diff
@@ -90,6 +90,7 @@ struct mount_info {
 	int deleted_level;
 	struct list_head deleted_list;
 	struct mount_info *next;
+        struct list_head tail;
```

Comment:
if you add this `struct list_head` member here, then you don't need to have `struct mount_info *next` member anymore as you can use `tail.next` to access to the next `struct mount_info` pointer, isn't it?